lab_1/main.py

```diff
@@ -105,7 +178,14 @@ def write_to_file(path_to_file: str, content: list):
     """
     Writes the result in a file
     """
-    pass
+    concordance_text = []
```

Comment:
here you just implement write function and then use it in start.py, not in this function directly